potassium/status.py

```diff
@@ -0,0 +1,97 @@
+from enum import Enum
+import time
+from typing import List, Tuple
+from dataclasses import dataclass
+
+from .types import RequestID
```

Comment:
nit: I'd organize imports (applies everywhere if you want to do it) - e.g. separate from & imports.

https://peps.python.org/pep-0008/#imports

> Imports should be grouped in the following order:

> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
You should put a blank line between each group of imports.